src/com/shmuelzon/HomeAssistantFloorPlan/Controller.java

```diff
@@ -892,9 +892,12 @@ private void generateSensorEntities(Map<String, Entity> homeAssistantEntities) {
     private boolean doStateIconsIntersect(Entity first, Entity second) {
         final double STATE_ICON_RADIUS_INCLUDING_MARGIN = 20.0;
 
-        double x = Math.pow(first.position.x - second.position.x, 2) + Math.pow(first.position.y - second.position.y, 2);
+        double centerDistance = Math.sqrt(
+                Math.pow(first.position.x - second.position.x, 2) + Math.pow(first.position.y - second.position.y, 2)
+        );
```

Comment:
If you feel compelled to refactor this function, please leave this as a single line.

Reply:
I'm not really compelled, but IMHO using the common formula makes it more clear on what's happening.

Without line-wrapping, the line would be very long.
I'm used to style-guides which prohibits the length exceeding an x amount of characters, hence the line-wrapping.
IMHO a line should not exceed 120 characters for better readability.
If you're not a fan of applying style guides, I guess I have to change it back to a single line of 146 characters.

Reply:
In the olden days, I was actually accustom to lines of 80 characters and still maintain that for C code. I don't feel as obligated to do that with Java, for some reason. In any case, as there is no defined style guide nor any tool to enforce that, I generally break lines where it would help to understand what's going on. In this specific example, the actual formula isn't really important for anyone maintaining the code, all you need to know is that it calculates the distance and you've done that by renaming the variable.